packages/definitions-parser/src/git.ts

```diff
@@ -61,12 +74,24 @@ export function gitChanges(
     const dep = getDependencyFromFile(diff.file);
     if (dep) {
       const key = `${dep.typesDirectoryName}/v${dep.version === "*" ? "*" : formatTypingVersion(dep.version)}`;
-      addedPackages.set(key, [dep, diff.status]);
+      if (diff.status === "R") {
+        addedPackages.set(key, [dep, "A"]);
+        const srcDep = getDependencyFromFile(diff.source);
+        if (srcDep) {
+          const srcKey = `${srcDep.typesDirectoryName}/v${
+            srcDep.version === "*" ? "*" : formatTypingVersion(srcDep.version)
+          }`;
+          addedPackages.set(srcKey, [srcDep, "D"]);
+        }
+      } else {
+        addedPackages.set(key, [dep, diff.status]);
+      }
     } else {
+      const status = diff.status === "A" || diff.status === "R" ? "add" : "delete";
```

Comment:
```suggestion
      const status = diff.status === "A" || diff.status === "D" ? "add" : "delete";
```

I think this is a typo? This else branch can't have `R`, right?

Reply:
The else branch just above this one can't; this is the else branch when the file being moved outside the monorepo structure. That is, for files that's don't match the pattern `types/X/file.ts`

Reply:
Oh oops, I misread this as `A -> add, R -> delete`, not `A or R -> add, else delete`.